streams.yml

```
@@ -69,8 +69,7 @@ rhel8:
 rhel9:
   # the most recent release at present. since we yum update this, it does not need to float.
   # it is important that we not build from unreleased builds and publish them.
-  image: registry-proxy.engineering.redhat.com/rh-osbs/rhel-els@sha256:c00fe5ad75658431c97b3a225db5625a97c8f3a28253af7ec3aa622db235ff64
-  # ^^ rhel-els-container-9.0.0-974
+  image: registry-proxy.engineering.redhat.com/rh-osbs/rhel-els:9.0.0-1228
```

I have this weird hangup about using immutable shasums rather than mutable tags :)

```suggestion
  # rhel-els-container-9.0.0-1228
  image: registry-proxy.engineering.redhat.com/rh-osbs/rhel-els@sha256:47aa1b5572e41f79537afce517c7edbad6ae592f7ab819b1ce9e7d22293ef0ff
```